src/oas.md

```diff
@@ -2126,20 +2128,20 @@ requestBody:
         type: object
         properties:
           # default content type for a string without `contentEncoding`
-          # is `text/plain`
+          # is "text/plain"
           id:
             type: string
             format: uuid
 
           # default content type for a schema without `type`
-          # is `application/octet-stream`
+          # is "application/octet-stream"
           profileImage: {}
 
           # for arrays, the `encoding` field applies the Encoding Object
           # to each item individually and determines the default content type
           # based on the type in the `items` subschema, which in this example
           # is an object, so the default content type for each item is
-          # `application/json`
+          # "application/json"
```

Comment:
Same with these and other changes from backticks to quotes on media types.  We should either keep the regular backticks formatting in the comments for everything that is formatted that way, or for nothing.